### DISCUSSION TITLE
Why has downloadhelper stopped working today on an update

### DISCUSSION THREAD

**Discussion by steeltoys** [aclap-dev/video-downloadhelper#101] [Help and Q&A]
### Your question

The software has worked perfect until sometime this afternoon when it was updated to the latest version 8.2,0.23. I used to simply click on the icon and select the file either HLS or a file in mb select download and hey presto it downloaded . Nothing works now so how can this be an improved version, I wont the Jan 2024 vesrion back as it worked with no issues - how do I do this?

steeltoys

**Comment by 701yekeparG** [comment]
The same here! The "Copy Url"-function doesn't work anymore! To the developer of this AddOn: Fix it, or most people won't use your AddOn anymore, because this function is the reason, why most of them use it!

**Reply by rocky1949** [reply]
Judging from how these reptiles operate, good luck.  I had a paid license and then it went poof and there is no way to recover it.  They aren't good human beings.

**Reply by grigoris79** [reply]
Same here guys.
What is happening?
I believe the last update f... everything.

**Reply by paulrouget** [reply]
> They aren't good human beings

I appreciate the insult.

I'm working on fixing the latest bugs.

**Reply by pitsi** [reply]
As a user who mostrly uses the copy url function, I am adding myself to this part of the discussion. I will try downgrading it to the previous version later today and report back. Ff 122.0.1 on debian x64 testing/unstable, downloadhelper 8.2.0.23. 

p.s. If the dev does read all these, PLEASE open the issues so that each broken feature can have its own issue and not list all of them in discussions of many posts. Thank you in advance.

---edit
I downgraded to 8.1.4.0 and works like it is supposed to. I disabled its automatic updates until all the issues of 8.2.x are resolved.

**Reply by pitsi** [reply]
Is it just me or the beta version of the addon is older than the current stable? I downloaded the xpi file from your page, installed it on ff and it is at 8.2.0.21a1, which has the same issues stable has. Is there anything newer?

Tbh, I would not mind a beta released addon, provided that it auto-updates and that I can report any issues in a sane manner. And that is not github's discussions, because someone can easily lose track with all the messages and the replies below each one! :(

**Reply by paulrouget** [reply]
Double check, the beta should be 8.2.1 now.

**Reply by pitsi** [reply]
Thank you for updating the beta file (for firefox).
I noticed that 8.2.1 reached the chrome and edge stores as stable, so I assume 8.2.1 is on its way for firefox as well.

**Reply by pitsi** [reply]
Time to report that 8.2.1 beta works properly on ff. Well done :)

**Reply by 701yekeparG** [reply]
How did you disabled automatic updates? There is no setting for that!

**Reply by 701yekeparG** [reply]
Oh sorry, you meant the auto update settings from the addons in general?? I just found them!

**Comment by alvereto** [comment]
Il software ha funzionato perfettamente fino a quando è stato aggiornato all'ultima versione 8.2,0.23. Di solito facevo semplicemente clic sull'icona e selezionavo il file HLS o un file in MB, seleziona download e presto lo scaricavo. Ora non funziona nulla, quindi come può essere una versione migliorata? Se scarica il file questo non si vede, tra l'altro ho anche pagato l'abbonamento!

**Comment by kchew534** [comment]
Same here.

I used to download audio files with no problem, but after the update today I got this error

SideDownload error:

[mp3 @ 0x11270a380] Invalid audio stream. Exactly one MP3 audio stream is required.
[out#0/mp3 @ 0x1127066a0] Could not write header (incorrect codec parameters ?): Invalid argument
[aost#0:0/copy @ 0x12266b080] Error initializing output stream: 


I noticed that there is a new option "Audio-only download", it requires me to buy a license to use the converter. I don't mind to dole out $28 but want to make sure it is not a bug in the application.

**Reply by rocky1949** [reply]
I already paid for mine once.  Not going to do it again.

**Comment by a-f-zz** [comment]
Version 8.1.4.0 worked great. Version 8.2.0.23 sucks. Stop updating without my permission and breaking stuff. I WANT 8.1.4.0 BACK!

**Reply by paulrouget** [reply]
What exactly broke?

**Reply by grigoris79** [reply]
Just everything and mostly the copy link.
Just READ the to many post that mention AFTER the last update the addon simply DOESN'T WORK CORRECTLY.

How we can return to previews version of the firefox addon???

**Reply by paulrouget** [reply]
The copy link is a known bug. I'm working on it.

> Just READ the to many post that mention AFTER the last update the addon simply DOESN'T WORK CORRECTLY.

Please understand that it's also working for million of other people. I need a specific link that's not working for you.

**Comment by aj3754** [comment]
Hmm. Doesn't work anymore either. Here is what I get:

https://ok6-31.vkuser.net/?expires=1708141951986&srcIp=23.111.78.42&pr=10&srcAg=GECKO&ms=95.142.206.35&type=1&sig=ukXDjDN_V2o&ct=6&urls=45.136.21.26%3B185.226.52.4&clientType=0&zs=65&id=5998420822538:-1: error : unterminated entity reference srcIp=23.111.78.42&pr=10&srcAg=GECKO&ms=95.142.206.35&type=3&sig=iWZE03KcZaI&ct=12&urls=45.136.21.26%3B185.226.52.4&clientType=0&zs=65&id=5998420822538
[dash @ 0x5eb9c00] Error when loading first fragment of playlist
[in#0 @ 0x5eb9ac0] Error opening input: Invalid data found when processing input
Error opening input file https://ok6-31.vkuser.net/?expires=1708141951986&srcIp=23.111.78.42&pr=10&srcAg=GECKO&ms=95.142.206.35&type=1&sig=ukXDjDN_V2o&ct=6&urls=45.136.21.26%3B185.226.52.4&clientType=0&zs=65&id=5998420822538.
Error opening input files: Invalid data found when processing input

**Reply by paulrouget** [reply]
From what website did you try downloading?

**Reply by aj3754** [reply]
https://ok.ru/video/7055973288458
It's some soccer highlights.

**Reply by paulrouget** [reply]
Appreciated. Thank you.

**Reply by Mattmjs** [reply]
@aj3754 . here is a tip for you while you wait for a fix. Make your link a mobile link by adding **m.**

https://m.ok.ru/video/7055973288458

**Reply by aj3754** [reply]
Thanks for the tip. I will try that next time it doesn't work. In the meantime I was able to download it with yt-dlp (a fork of youtube-dl).

**Comment by paulrouget** [comment]
Hey, I need the url of the page you're trying to download from. Otherwise I can't help.

**Reply by grigoris79** [reply]
Is NOT only one url.
BEFORE the update work fine AFTER the update doesn't. 

Is so simple.

**Reply by paulrouget** [reply]
I understand. But please give me one URL that's not working please. As it's working for many people too. I'm trying to figure this out.

**Comment by paulrouget** [comment]
**Reminder: be a decent human being, and do not insult developers. I understand the update broke stuff, and I'm working on figuring out what broke. But please understand that insults do not help.**

**Reply by rocky1949** [reply]
If you had an actual link, a direct contact for help I just might think better of you.  Doesn't have a good look to it when someone has paid for something and then can't contact a real person for help.  This isn't round about way of contact doesn't cut it.

**Reply by GrampaWildWilly** [reply]
You're not dealing with IBM, Microsoft, Google, or NVidia here.  You're dealing with an operation that consists of 3 people here, by my count.  And I'm not one of them.  I don't work for VDH.  I'm just another user.  This is NOT a round about way of contacting the developers.  Paul IS a developer.  He's here & he's active.  Be reasonable.  The amount you paid is a pittance, & with each passing year it will get cheaper because it is a LIFETIME license.  It's a ONE-TIME purchase.  FOR LIFE.  So I say again, be reasonable.  If you want results, you would do much better to endear yourself to Paul than to antagonize him.  It's so easy to hide behind an anonymous user ID & take potshots at somebody.  Learn some manners.

**Reply by rocky1949** [reply]
Steven Rocky Raybell, you want my address and phone number? "GrampaWildWilly"  Define Jerk.

**Reply by rocky1949** [reply]
Turns out that lifetime license isn't a "lifetime" license by the way.

**Reply by paulrouget** [reply]
@rocky1949 I understand the frustration, and I'm working on trying to fix the issue. Please give us some time to figure this out.

On the other hand: calling us names is not an acceptable behavior. Please be nice.

**Comment by grigoris79** [comment]
I don't believe allow to send links. How i can send you personal message?

I send you mail.

**Reply by paulrouget** [reply]
<snip>

**Comment by rocky1949** [comment]
Ordinarily I'm not nice when I feel I've been ripped off for a pittance. I fail to see how "fixing" the issue will get me my license without another payment after the message I got when trying to recover the license.  But bottom line I can do what I want to do without it, with the exception of just downloading and then converting without paying someone.  I would advise others to run malware because the loss of the conversion wasn't the only issue I ended up with.  I'm out of here.

    On Thursday, February 15, 2024 at 10:23:18 PM PST, Paul ***@***.***> wrote:  
 
 


@rocky1949 I understand the frustration, and I'm working on trying to fix the issue. Please give us some time to figure this out.

On the other hand: calling us names is not an acceptable behavior. Please be nice.

**Reply by jcfrog** [reply]
You never need to repurchase, we sell lifetime licenses, you just need to revalidate your key when needed : https://github.com/aclap-dev/video-downloadhelper/wiki#go-premium

**Comment by steeltoys** [comment]
Found an earlier version of the software 8,1.40 and installed it, now the videos actually download no issues. I found that the latest version realeased yesterday might have worked but it was very different a would only download 2 files before it locked up my firefox totally and became unresponsive - totally buggered. Why do you release these updates as working versions if they put users through hell. Do it as Beta only. I have now switched off auto updates aas I cant trust the software any more. I was looking t donating but after this episode I am not sure now.

steeltoys

**Reply by dhcernese** [reply]
where are the earlier kits?  i can't find them.

**Reply by jcfrog** [reply]
please read this: https://github.com/aclap-dev/video-downloadhelper/wiki/Older-versions

**Reply by televischn** [reply]
Thanks for that hint!! 
8.1.4.0 could finally save me a lot of nerves and time.
(For Firefox users: https://addons.mozilla.org/en-US/firefox/addon/video-downloadhelper/versions/ )

**Comment by amiga-500** [comment]
> where are the earlier kits? i can't find them.

For Firefox there is a version history link where u can download and install prior versions.  That is how i solve this ongoing broken extension issue.  Right now i had to downgrade a few versions to make it work again.  I've officially turned off auto-update because it seems this problem could be permanent for me. Ahh well.  Was fun while it lasted :(

**Comment by alvereto** [comment]
Ho scaricato alcuni video e ha smesso di scaricare, scarica un video della stessa lunghezza ma completamente nero. Faccio presente che ho pagato e non sto usufruendo di niente. Siete dei truffatori.

Video DownloadHelper
Versione 8.2.1.1 (stable)
Lingua del browser: it
Versione di produzione
Compilato il 2024-02-19
Opzioni di versione: browser=chrome, linuxlic=false, noyt=true, target=google
Piattaforma Win x86-64
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36
Versione app integrativa: VdhCoApp 2.0.19
Percorso app integrativa: C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe

**Reply by Mattmjs** [reply]
No one is scamming you, you say the videos are completely black. It sounds like the platform your using is protecting their
content. Called DRM, it will give an unusable file should you try download it. This is not the fault of the software.
Request a refund if your not satisfied.

**Reply by GrampaWildWilly** [reply]
Agreed.  If you want meaningful help, post at least 1 URL of a video that is giving you trouble.

**Comment by priteshbbbb** [comment]
Not sure if this is an update problem - but now when I use CopyUrl -- instead of it getting the actual video names I get deprecatedName, meaning I can't use it for bulk downloads anymore (as all the videos have the same name...)

Reason I think it may be unrelated to the most recent update is that I installed older versions in Firefox and it still behaves the same way.

It does this in both Chrome and Firefox.
Any advice?
Thanks for all your hard work, guys!

**Comment by grigoris79** [comment]
Still wait for an update to fix this issue.......

**Reply by paulrouget** [reply]
Be patient, Mozilla and Google are reviewing the addon. 8.2.1 should come soon.

**Comment by grigoris79** [comment]
How we can download the beta version?
And the final version of this addon is gonna be late?

**Reply by jcfrog** [reply]
The version is still waiting for approval, it won't be long.
Betas are here : https://github.com/aclap-dev/video-downloadhelper/wiki/Beta-versions

**Comment by grigoris79** [comment]
Why the approval is so late really??????
And if we install the beta when the stable approved does it updated to stable from beta?

**Reply by jcfrog** [reply]
ask Mozilla :)
there is nothing we can do but wait.
There is always a beta released with the stable, you can come back to stable by just reinstalling it once.

**Comment by grigoris79** [comment]
So you must RE INSTALL THE ADDON???
DOESN'T SIMPLY. UPDATED AS ALL ADDON AFTER ANY BETA RELEASE???

On 21 Φεβρουαρίου 2024 21:46:46 jcfrog ***@***.***> wrote:
>
> ask Mozilla :)
> there is nothing we can do but wait.
> There is always a beta released with the stable, you can come back to 
> stable by just reinstalling it once.—
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you commented.Message ID: 
> ***@***.***>


--18dcd4240c52b6c28add62090e
Content-Type: text/html; charset="UTF-8"
Content-Transfer-Encoding: quoted-printable

<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.=
w3.org/TR/html4/loose.dtd"><html>
<head>
<meta http-equiv=3D"Content-Type" content=3D"text/html; charset=3Dutf-8"></=
head>
<body>
<div dir=3D"auto">
<div dir=3D"auto">So you must RE INSTALL THE ADDON???&nbsp;</div><div dir=
=3D"auto">DOESN'T SIMPLY. UPDATED AS ALL ADDON AFTER ANY BETA RELEASE???&nb=
sp;</div><div dir=3D"auto"><br></div>
<div id=3D"aqm-original" style=3D"color: black;">

<div class=3D"aqm-original-body">
<div style=3D"color: black;">
<p style=3D"color: black; font-size: 10pt; font-family: sans-serif; margin:=
 8pt 0;">On 21 =CE=A6=CE=B5=CE=B2=CF=81=CE=BF=CF=85=CE=B1=CF=81=CE=AF=CE=BF=
=CF=85 2024 21:46:46 jcfrog ***@***.***&gt; wrote:</p>
<blockquote type=3D"cite" class=3D"gmail_quote" style=3D"margin: 0 0 0 0.75=
ex; border-left: 1px solid #808080; padding-left: 0.75ex;">
<p></p>
<p dir=3D"auto">ask Mozilla :)<br>
there is nothing we can do but wait.<br>
There is always a beta released with the stable, you can come back to stabl=
e by just reinstalling it once.</p>

<p style=3D"font-size:small;-webkit-text-size-adjust:none;color:#666;">=E2=
=80=94<br>Reply to this email directly, <a href=3D"https://github.com/aclap=
-dev/video-downloadhelper/discussions/101#discussioncomment-8548103">view i=
t on GitHub</a>, or <a href=3D"https://github.com/notifications/unsubscribe=
-auth/AH5EPPAAHJ4GL45EB4PVLELYUZFKJAVCNFSM6AAAAABDKW6HWSVHI2DSMVQWIX3LMV43S=
RDJONRXK43TNFXW4Q3PNVWWK3TUHM4DKNBYGEYDG">unsubscribe</a>.<br>You are recei=
ving this because you commented.<img src=3D"https://github.com/notification=
s/beacon/AH5EPPC6DMRLYI6VNMC6I63YUZFKJA5CNFSM6AAAAABDKW6HWSWGG33NNVSW45C7OR=
4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQAQJXQO.gif" height=3D"=
1" width=3D"1" alt=3D""><span style=3D"color: transparent; font-size: 0; di=
splay: none; visibility: hidden; overflow: hidden; opacity: 0; width: 0; he=
ight: 0; max-width: 0; max-height: 0; mso-hide: all">Message ID: <span>&lt;=
aclap-dev/video-downloadhelper/repo-discussions/101/comments/8548103</span>=
<span>@</span><span>github</span><span>.</span><span>com&gt;</span></span><=
/p>
</blockquote>
</div>
</div>


</div><div dir=3D"auto"><br></div>
</div></body>
</html>

--18dcd4240c52b6c28add62090e--

**Reply by GrampaWildWilly** [reply]
The public general availability version is in one channel.  The beta version is in another channel.  You can't be in both channels at the same time.  You get into one channel or the other simply by installing either the general availability version or the beta version.  You can switch channels by installing the version from the other channel.  You can do that whenever you like & as often as you like.

Automatic update is a bad idea.  Some users reported (on the old Google Groups forum) that the browser unilaterally decided to update VDH while they happened to have downloads in progress.  That killed the downloads, necessitating a redo of the downloads from scratch.  You have to understand that the browser updates VDH.  The VDH software itself does not update VDH.  The VDH developers do not update VDH on your system.  Your browser performs the update based on settings you have specified in the browser settings.  Browser.  You need to understand that.  The browser updates VDH.  So you have to go into the browser settings & tell the browser it's not allowed to automatically update VDH.  Other extensions are probably good candidates for automatic updating.  VDH is not because of this potential unexpected interruption of your normal operation.  When you turn off automatic updates, the browser displays a new button for allowing you to manually check for an update when it would be convenient for you to install the update.  You can then decide to perform the update or not perform it.  But updates now can occur only on your initiative.  You have to remember to go check.  Paul has said he is working on fixing that but I gather the timeline will be a bit long.  So don't hold your breath waiting for that.  Turn off automatic update now.  Right now.  Don't wait.  Do it now.
